### PR TITLE
uses fcrepo_wrapper to provide Fedora 4.7 to dev environment fixes #1034

### LINKED ISSUE #1034: Upgrade Fedora to current release
Spike to try upgrading Fedora to the current version (4.5) and determine how much work will be needed on the rest of the stack to make it work. Our aim is to catch up to the current Fedora if possible, but if the work involved is too big we may decide to make the Fedora upgrade part of the PCDM migration process (so as to avoid redundant work making our current codebase work with Fedora 4.5 and then throw it all out again when we move on to the PCDM work).

Replaces #293, #465, #466

### PR DESCRIPTION
**Pull request by pgwillia**
* fcr:export was deprecated
 * configured logging
 * updated reset-all and restart-all scripts
 * updated README

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -61,6 +61,7 @@ group :development, :test do
   gem "ruby-debug-passenger"
   gem "selenium-webdriver"
   gem "jettywrapper"
+  gem "fcrepo_wrapper"
```
Comment (murny) [inline Gemfile]: Don't know anything about this gem so bare with me,  do these rake tasks ever need to be ran in production environment? Does the wrapper only provide rake tasks/cli? if so you could potentially set it to `required: false` as it doesn't needed to be loaded in every process?

**Comment by murny** [comment]
Looks good :+1: ! Just one small question about the gemfile. Also how does this impact a developer? Do I now have to do `rake jetty:stop && rake fcrepo:stop` and `rake jetty:start && rake fcrepo:start`, to bounce fedora?

File: Gemfile (hunk 1 above)
Comment (pgwillia) [inline Gemfile]: These rake tasks never need to be ran in production. It's only rake tasks/cli, I believe. I'll make the change.

**Comment by pgwillia** [comment]
Use ``rake jetty:stop``` and ``rake jetty:start`` to bounce solr.  Use ```rake fcrepo:stop``` and ```rake fcrepo:start``` to bounce fedora.  Could that be clearer in the README?

**Comment by murny** [comment]
That makes sense! Nah README is fine! Just viewed it in it's entirety and makes much more sense now then just the small diff I was seeing.  Thanks!

**Comment by pbinkley** [comment]
When should we merge this? I assume we don't want to be developing on 4.7 until the production environment is ready. Do we want to keep this on a branch until then?

**Comment by pbinkley** [comment]
Obsolete, so closing.

**Pull request closed without merging**